addOns/fuzz/src/main/resources/org/zaproxy/zap/extension/fuzz/resources/Messages.properties

```diff
@@ -371,7 +370,6 @@ fuzz.payload.processor.sha512Hash.description = Using encoding ''{0}''
 
 fuzz.payload.processor.trim.name = Trim
 fuzz.payload.processor.trim.description = To {0} characters
-fuzz.payload.processor.expand.length.label = Length:
```

Comment:
`expand` → `trim` and update in `TrimStringProcessorUIPanel`, or the key renamed to be generic.